.github/CODEOWNERS

```diff
@@ -81,6 +81,9 @@
 # PRLabel: %Search
 /sdk/search/ @xirzec @sarangan12
 
+/sdk/applicationinsights/applicationinsights-query/ @divya-jay @geneh @alongafni
+/sdk/operationalinsights/loganalytics/ @divya-jay @geneh @alongafni
```

Comment:
Recommend you take the entirety of ```/sdk/applicationinsights``` and ```/sdk/operationalinsights``` so that you can be kept across any other changes that may impact your libraries (some things fall outside the particular package directories.

Reply:
Should it be the case even if other directories are not owned by the team?

Reply:
The other directories are for the management packages. Assuming you are from the applicationinsights and operationalinsights teams, it would be in your interest to be aware of when the corresponding arm/management packages get updated. So, I would agree with @mitchdenny